tests/unit/style/parsing/background.rs

```diff
@@ -0,0 +1,125 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use cssparser::Parser;
+use media_queries::CSSErrorReporterTest;
+use style::parser::ParserContext;
+use style::properties::longhands::{background_attachment, background_clip, background_color, background_image};
+use style::properties::longhands::{background_origin, background_position, background_repeat, background_size};
+use style::properties::shorthands::background;
+use style::stylesheets::Origin;
+use url::Url;
```

Comment:
This will have to be `servo_url::ServoUrl` (and youll have to replace occurrences of `Url` below with `ServoUrl`). URL handling for style was changed since you made this pull request, which is why Travis CI is failing.

Please rebase over master and make this change (let me know if you need help!)